examples/training/object_detection_3d/waymo/point_pillars.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import time
+
+import tensorflow as tf
```

Comment:
nit: object_detection_3d/wod/point_pillars.py?

Reply:
Since the dataloader is under `keras_cvdataloaders/waymo/`, IMO it's better to call this `waymo` for consistency.

But maybe we should change both to `wod`?